ocaml/xapi/xapi_vdi.ml

```diff
@@ -22,49 +22,49 @@ open D
 (**************************************************************************************)
 (* current/allowed operations checking                                                *)
 
+let feature_of_op =
```

Comment:
IMO this function should go to `Smint` but I think it is ok to keep it here for this PR for clarity as it was here.